infra/gcp/main.tf

```diff
@@ -59,3 +59,12 @@ resource "generic_local_cmd" "kubeconfig" {
     EOT
   }
 }
+
+# Synchronises local DAG files with those in the Cloud Composer environment.
```

Comment:
To me, it rather seems that you are synchronizing the DAG files in the Cloud Composer env with those in your local env ?